apps/term-apply/go.mod

```diff
@@ -11,11 +11,15 @@ require (
 	golang.org/x/crypto v0.0.0-20220321153916-2c7772ba3064
 )
 
+require github.com/aws/aws-sdk-go v1.43.44
```

Comment:
Whats the reasoning behind having this in a separate `require` instead of in the above block? More curious as to whats happening here than anything else

Reply:
good question, i didn't even notice, that file is managed by `make deps`. It could be because I didn't rm `go.mod` and `go.sum` first.

Reply:
100% agree, I opened `inf 515` when it go too unwieldy to do as part of this PR.

Reply:
I had to know. I think go mod just got goofy because of the 1.16 to 1.17 upgrade (some of the deps were picked by 1.16 and I was u. I was able to reproduce the messages below. I went ahead and recreated the files, we got a quick dep bump out of it.

```
$ make deps
go mod verify
all modules verified
go mod tidy
github.com/nebulaworks/orion/apps/term-apply/pkg/s3file imports
        github.com/aws/aws-sdk-go/service/s3 imports
        github.com/aws/aws-sdk-go/aws/awsutil imports
        github.com/jmespath/go-jmespath tested by
        github.com/jmespath/go-jmespath.test imports
        github.com/jmespath/go-jmespath/internal/testify/assert imports
        github.com/davecgh/go-spew/spew loaded from github.com/davecgh/go-spew@v1.1.0,
        but go 1.16 would select v1.1.1
github.com/nebulaworks/orion/apps/term-apply/pkg/s3file imports
        github.com/aws/aws-sdk-go/service/s3 imports
        github.com/aws/aws-sdk-go/aws/awsutil imports
        github.com/jmespath/go-jmespath tested by
        github.com/jmespath/go-jmespath.test imports
        github.com/jmespath/go-jmespath/internal/testify/assert imports
        gopkg.in/yaml.v2 loaded from gopkg.in/yaml.v2@v2.2.8,
        but go 1.16 would select v2.3.0

To upgrade to the versions selected by go 1.16:
        go mod tidy -go=1.16 && go mod tidy -go=1.17
If reproducibility with go 1.16 is not needed:
        go mod tidy -compat=1.17
For other options, see:
        https://golang.org/doc/modules/pruning
make: *** [GNUmakefile:12: deps] Error 1

$ go mod tidy -go=1.16 && go mod tidy -go=1.17

$ make deps
go mod verify
all modules verified
go mod tidy
```

Reply:
Ok well im glad I wasnt missing something, glad you dug into it!